modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -29,3 +32,70 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
 function oe_content_publication_post_update_00002() {
   \Drupal::service('module_installer')->install(['oe_content_documents_field']);
 }
+
+/**
+ * Create the new fields in the Publication content type.
+ */
+function oe_content_publication_post_update_00003(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00003_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update field labels.
```

Comment:
It's not only doing that, it also sets fields translatable. Could we move this update in 00003 where the new fields are created?

Reply:
00003 is used to create new fields.
Might be better to adjust function description? Instead of `Update field labels.` use `Update field settings.`?

Reply:
I updated the function description.